docs/content/en/docs/cluster-operator/vms-deploy.md

```diff
@@ -198,6 +198,53 @@ kube-system   nodelocaldns-fpfxj                 1/1     Running   0          8m
 
 We can see that the cluster on VMs is installed successful.
 
+## Cluster upgrading
```

Comment:
Since you are talking about all the lifecycle of cluster, the whole doc title is not appropriate

If you want to keep the name, you can separate different docs